resources/sql_servers.go

```diff
@@ -97,19 +116,228 @@ func SQLServers() *schema.Table {
+				Resolver:    fetchSqlServerAdmins,
+				Columns: []schema.Column{
+					{
+						Name:        "server_id",
+						Description: "Unique ID of azure_sql_servers table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "administrator_type",
+						Description: "Type of the sever administrator",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("AdministratorProperties.AdministratorType"),
+					},
+					{
+						Name:        "login",
+						Description: "Login name of the server administrator",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("AdministratorProperties.Login"),
+					},
+					{
+						Name:        "sid",
+						Description: "SID (object ID) of the server administrator",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.PathResolver("AdministratorProperties.Sid"),
+					},
+					{
+						Name:        "tenant_id",
+						Description: "Tenant ID of the administrator",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.PathResolver("AdministratorProperties.TenantID"),
+					},
+					{
+						Name:        "azure_ad_only_authentication",
+						Description: "Azure Active Directory only Authentication enabled",
+						Type:        schema.TypeBool,
+						Resolver:    schema.PathResolver("AdministratorProperties.AzureADOnlyAuthentication"),
+					},
+					{
+						Name:        "resource_id",
+						Description: "Resource ID",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("ID"),
+					},
+					{
+						Name:        "name",
+						Description: "Resource name",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "type",
+						Description: "Resource type",
+						Type:        schema.TypeString,
+					},
+				},
+			},
 		},
 	}
 }
 
 // ====================================================================================================================
 //                                               Table Resolver Functions
 // ====================================================================================================================
-func fetchSqlServers(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan interface{}) error {
+func fetchSqlServers(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan interface{}) error {
 	svc := meta.(*client.Client).Services().SQL.Servers
 	servers, err := svc.List(ctx)
 	if err != nil {
 		return err
 	}
-	res <- *servers.Value
+	for servers.NotDone() {
+		res <- servers.Values()
+		if err := servers.NextWithContext(ctx); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func fetchSqlServerPrivateEndpointConnections(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan interface{}) error {
+	server := parent.Item.(sql.Server)
```

Comment:
I advise to add the type assertion check 
i,ok := x.(y)

Reply:
Fixed, thank you!